Source/AwsCommonRuntimeKit/sdkutils/endpoint/EndpointsRequestContext.swift

```diff
@@ -4,7 +4,7 @@
 import AwsCSdkUtils
 
 /// Request context used for resolving endpoint
-public class CRTAWSEndpointsRequestContext {
+public class EndpointsRequestContext {
```

Comment:
Seems like you were keeping the "AWS" prefix on classes that were AWS-specific like `AWSCredentials`, `AWSSigner`, `AWSProfile`, etc

but endpoints is also just an AWS thing and you removed the prefix ???

Reply:
Thanks, fixed.